provider/go.mod

```diff
@@ -5,6 +5,6 @@ go 1.16
 replace github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20211230170131-3a7c83bfab87
 
 require (
-	github.com/Twingate-Labs/terraform-provider-twingate v0.1.7-0.20220429123622-020282518e12
+	github.com/Twingate/terraform-provider-twingate v0.1.7-0.20220429170903-fbd2e621164b // indirect
```

Comment:
Hi @ekampf only glancing quickly here but how is this supposed to work because in the official repo you haven't (still) released a 0.1.7. release and that was supposed to happen last Thursday.